app/services/doc_auth/mock/result_response.rb

```diff
@@ -71,6 +71,39 @@ def attention_with_barcode?
         parsed_alerts == [ATTENTION_WITH_BARCODE_ALERT]
       end
 
+      def self.create_image_error_response(status)
+        error = case status
+                when 438
+                  Errors::IMAGE_LOAD_FAILURE
+                when 439
+                  Errors::PIXEL_DEPTH_FAILURE
+                when 440
+                  Errors::IMAGE_SIZE_FAILURE
+                end
+        errors = { general: [error] }
+        message = [
+          'Unexpected HTTP response',
+          status,
+        ].join(' ')
+        exception = DocAuth::RequestError.new(message, status)
+        DocAuth::Response.new(
+          success: false,
+          errors: errors,
+          exception: exception,
+          extra: { vendor: 'Acuant' },
+        )
+      end
+
+      def self.create_network_error_response
+        errors = { network: true }
+        DocAuth::Response.new(
+          success: false,
+          errors: errors,
+          exception: Faraday::TimeoutError.new,
+          extra: { vendor: 'Acuant' },
```

Comment:
As I said on line 103, I think it could be good to indicate that the vendor is a mock.